codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/localservice/shapevisitor/DafnyToSmithyShapeVisitor.java

```diff
@@ -349,17 +350,59 @@ public String doubleShape(DoubleShape shape) {
     @Override
     public String unionShape(UnionShape shape) {
         writer.addImportFromModule("github.com/dafny-lang/DafnyRuntimeGo", "dafny");
-        return """
```

Comment:
Why these are showing up in Diff? These changes are not on the branch. I suspect something is going on with your merge/pull/push. Can you double check?

Reply:
I reverted back all the merged commits and merged again. This worked.